explorer/src/lib/chain-info/addCountAndUnique.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Returns an array of unique objects, sorted by the frequency of their occurrence, with an additional count property representing the number of times that city-country combination appeared in the input data.
```

Comment:
Could we split this comment over several lines? Not sure why the prettier hasn't done it (as we should have limit for the chars per line) – I assume comments are excluded?